app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -124,8 +124,9 @@ def virt_who_output(format = nil)
 EOF
   if [ $? -ne 0 ]; then result_code=$(($result_code|#{error_code(:virt_who_config_file_issue)})); fi
 
-  step 4 "Creating sysconfig virt-who configuration"
-  cat > #{sysconfig_file_path} << EOF
+  step 4 "Creating sysconfig virt-who configuration and removing old configuration file if present"
+  rm -f #{old_config_path}
```

Comment:
Is it necessary that we do this cleanup? I'd expect the new version of virt-who to be doing the cleanup, but in general not sure if that's our responsibility. Just being cautious :)

Reply:
Leaving the file on the system won't hurt anything, I was just trying to make sure we cleaned up after ourselves. I can instead of removing it, put a message to the user to manually remove it, or I can file a BZ for virt-who to check and make sure the file is removed if the path is changed.

Reply:
I think I'm OK with the cleaning. You may wanna check with Will first to make sure we aren't overlooking anything

Reply:
@chris1984 any word one way or the other from Will ?

Reply:
me-- I forgot. I'll email him today.